app/bundles/LeadBundle/Entity/StagesChangeLog.php

```diff
@@ -44,6 +44,7 @@ public static function loadMetadata(ORM\ClassMetadata $metadata): void
 
         $builder->setTable('lead_stages_change_log')
             ->setCustomRepositoryClass(StagesChangeLogRepository::class);
+            ->addIndex(['date_added'], 'lead_stages_change_log_date_added');
```

Comment:
`PHP Parse error:  syntax error, unexpected token "->" in /home/runner/work/mautic/mautic/app/bundles/LeadBundle/Entity/StagesChangeLog.php on line 47`
Remove the semicolon on the previous line.

Reply:
Done